### ISSUE TITLE
grid_v, grid_zones_v, and polygrid_v: which one is used for area calculation?

### ISSUE THREAD

**Issue by jdduh** [PSU-CSAR/vb-bagis-h#32]
The specific attribute fields I refer to are "AREA_SqKm" and "AREA_Acre." These two fields should be calculated on the grid_v layer. However, I believe some BAGIS-H post processing tools end up calculate the attributes on polygrid_v. I have yet to figure out which tool is the culprit. Could you search for the subroutine(s) that perform the AREA calculations and make sure that they calculate on the grid_v vector layer? It might be the eliminate or the assign sequential HRU ID tool. I also had changed the "allow non-contiguous HRU" setting several times during the refinement process.

**Comment by lbross** [comment]
It is not grid_zones_v because that is managed by BAGIS-P. What is the symptom? Are these 2 fields missing from grid_v? Or are the values incorrect?

**Comment by jdduh** [comment]
The grid_v of the HRU that I created with the Eliminate tool doesn't have these area fields. Instead, only the polygrid_v has these area fields. The values look correct.

**Comment by lbross** [comment]
I was not able to replicate this behavior. My grid_v always had the area fields. For non-contiguous hrus, grid_v is copied to polygrid_v. Then polygrid_v id dissolved on the hru_id to create the non-contiguous hrus. I did notice that we weren't recalculating the area fields after the dissolve which is bad. I fixed this. This should also add the fields if they are missing for some reason. This will be in the next BAGIS-H release.
